woodwork/column_accessor.py

```diff
@@ -108,32 +108,29 @@ def schema(self):
         return copy.deepcopy(self._schema)
 
     @property
+    @_check_schema_init("Series")
```

Comment:
storing either "Series" or the whole message in a constant seems the like a good idea